src/pykiso/lib/connectors/cc_socket_can/cc_socket_can.py

```diff
@@ -150,16 +150,12 @@ def _cc_close(self) -> None:
             del self.logger
             self.logger = None
 
-    def _cc_send(self, msg: MessageType, raw: bool = False, **kwargs) -> None:
+    def _cc_send(self, msg: MessageType, **kwargs) -> None:
         """Send a CAN message at the configured id.
 
-        If remote_id parameter is not given take configured ones, in addition if
-        raw is set to True take the msg parameter as it is otherwise parse it using
```

Comment:
I think the first line should  not be deleted. I saw above other places similar. Please make sure to only delete elements relative to this change :)